src/components/Microsoft.DotNet.UpgradeAssistant.MSBuild/ComponentIdentifier.cs

```diff
@@ -17,7 +17,14 @@ public ProjectComponents GetComponents(IProject project)
             }
 
             var file = project.GetFile();
-            var components = file.IsSdk ? GetSDKProjectComponents(project, file) : GetOldProjectComponents(project, file);
+
+            // SDK-style projects can target .NET Framework and use GAC-referenced app models,
+            // so old project components are checked regardless of SDK status
+            var components = GetOldProjectComponents(project, file);
```

Comment:
Maybe rename it then as it feels weird keeping the `Old` in the name

Reply:
Good point. I'll rename it.